src/components/Restoration/BusinessName.vue

```diff
@@ -91,7 +91,7 @@ export default class BusinessName extends Mixins(CommonMixin, DateMixin, NameReq
   // Global actions
   @Action(useStore) setBusinessNameValid!: (x: boolean) => void
   @Action(useStore) setCorrectNameOption!: (x: CorrectNameOptions) => void
-  @Action(useStore) setNameRequest!: (x: string) => void
+  @Action(useStore) setNameRequest!: (x: NameRequestIF) => void
```

Comment:
This type here wasn't causing issues right? It's just a cleanup?

Reply:
It was, because the call to setNameRequest was passing an object instead of a string.